src/test/BidderWithEjector.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.4;
+
+import {DittoMachine, IERC721TokenEjector} from "../DittoMachine.sol";
+
+contract BidderWithEjector is IERC721TokenEjector {
+
+    uint256 public ejections;
+
+    DittoMachine immutable dm;
```

Comment:
I don't think we need to have the `dm` variable in ejectors (same with Bidder.sol).

Reply:
ah yeah, I'll push a commit to get rid of these